examples/SentTeensy/SentTeensy.ino

```diff
@@ -0,0 +1,41 @@
+
+// Connect data pin of sensor to 18 of the Teensy 4.x
+
+#include <SentTimerTeensyQuad.h>
+#include <MLX90372.h>
+
+SentTimerTeensyQuad sent(3, true, 18);
+MLX90372 mlx(sent);
+
+unsigned long lastTime;
+volatile float mlxPosition;
+
+void mlx_cb(uint16_t sentPos){
+  mlxPosition = float(sentPos) / 4096;
```

Comment:
i would like to see this calculation in the class MLX90372

Reply:
I considered where this belongs. The sensor is highly configurable and it's nearly impossible to know what the user wants at the driver level. Providing the raw measurement value seems like the best choice to me and forcing the user to a float also seemed wrong.